### PR TITLE
Expanded Issuing API calls tutorial

### PR DESCRIPTION
**Pull request by jpop32**
Also added making calls to subnet EVMs

### CODE REVIEW

File: docs/apis/avalanchego/apis/issuing-api-calls.md [hunk 1]
```diff
@@ -16,15 +20,111 @@ where
...
+To interact with subnet instances of the EVM via the JSON-RPC endpoint:
+
+```
+/ext/bc/[blockchainID]/rpc
+```
+
+where `blockchainID` is the ID of the blockchain running the EVM. So for example, the RPC URL for the Swimmer Network (a subnet that runs the Crabada game) running on a local node would be:
+
+```
+http://127.0.0.1/ext/bc/2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX/rpc
```
Comment (yulin-dong) [inline docs/apis/avalanchego/apis/issuing-api-calls.md:90]: is http optional or mandatory? I was wondering that we should add http and https in the doc.

File: docs/apis/avalanchego/apis/issuing-api-calls.md [hunk 2]
```diff
@@ -16,15 +20,111 @@ where
...
+```
+
+Or for the WAGMI subnet on te Fuji testnet:
+
+```
+http://127.0.0.1/ext/bc/2ebCneCbwthjQ1rYT41nhd7M76Hc6YmosMAQrTFhBq8qeqh6tt/rpc
+```
+
+#### Websocket EVM endpoints
```
Comment (yulin-dong) [inline docs/apis/avalanchego/apis/issuing-api-calls.md:99]:
```suggestion
#### Websocket EVM Endpoints
```

File: docs/apis/avalanchego/apis/issuing-api-calls.md [hunk 3]
```diff
@@ -16,15 +20,111 @@ where
...
+
+where `blockchainID` is the ID of the blockchain running the EVM. For example, to interact with the C-Chain's Ethereum APIs via websocket on localhost you can use:
+
+```
+ws://127.0.0.1:9650/ext/bc/C/ws
+```
 
-Each API’s documentation specifies what endpoint a user should make calls to in order to access the API’s methods.
+:::info
+When using the [Public API](../public-api-server.md) or another host that supports HTTPS, use `wss://` instad of `ws://`.
```
Comment (yulin-dong) [inline docs/apis/avalanchego/apis/issuing-api-calls.md:120]: We should say the same about http vs https somewhere in this doc.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/apis/issuing-api-calls.md (hunk 3 above)
Comment (jpop32) [reply docs/apis/avalanchego/apis/issuing-api-calls.md:120]: Will add.

File: docs/apis/avalanchego/apis/issuing-api-calls.md [hunk 4]
```diff
@@ -5,9 +5,13 @@ sidebar_position: 2
 
 This guide explains how to make calls to APIs exposed by Avalanche nodes.
 
-### Endpoint {#endpoint}
+## Endpoints {#endpoints}
```
Comment (yulin-dong) [inline docs/apis/avalanchego/apis/issuing-api-calls.md:8]: {#endpoints} is unneeded here, however, it does not hurt.

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**